pkg/cloudprovider/aws/instance.go

```diff
@@ -127,6 +127,10 @@ func (p *InstanceProvider) launchInstances(ctx context.Context, constraints *v1a
 				ResourceType: aws.String(ec2.ResourceTypeInstance),
 				Tags:         v1alpha1.MergeTags(ctx, constraints.Tags, map[string]string{fmt.Sprintf("kubernetes.io/cluster/%s", injection.GetOptions(ctx).ClusterName): "owned"}),
 			},
+			{
+				ResourceType: aws.String(ec2.ResourceTypeVolume),
+				Tags:         v1alpha1.MergeTags(ctx, constraints.Tags, map[string]string{fmt.Sprintf("kubernetes.io/cluster/%s", injection.GetOptions(ctx).ClusterName): "owned"}),
```

Comment:
Thoughts on extracting out the merge tags call? This has a nice side effect of shortening the line enough that you can collapse the lines elsewhere:
```
TagSpecifications: []*ec2.TagSpecification{
    { ResourceType: aws.String(ec2.ResourceTypeInstance), Tags: tags },
    { ResourceType: aws.String(ec2. ResourceTypeVolume), Tags: tags },
}
```

Reply:
LGTM